### PR TITLE
Added Series.py

### PR DESCRIPTION
**Pull request by zyang888**
Please take a look at my Series.py. Thanks a lot!

### CODE REVIEW

**Comment by zyang888** [comment]
Please take a look at my SlicingLabExercise.py. Thanks a lot!

**Comment by zyang888** [comment]
Please take a look at my list_lab.py. Thanks a lot!

File: students/Zhengtang_Yang/GridPrinterExercise.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+print_grid(3)
+
+print('print_grid(11)')
+print_grid(11)
+
+print('print_grid(15)')
+print_grid(15)
+
+print('print_grid2(n,m)')
+def print_grid2(n,m):
```
Comment (mdgregor) [inline students/Zhengtang_Yang/GridPrinterExercise.py]: The goal with grid2 was use the first value to make it a n by n grid with the second value being the size of the box. So an entry of `print_grid2(5,3)` would result in:
```
+ - - - + - - - + - - - + - - - + - - - +
|       |       |       |       |       |
|       |       |       |       |       |
|       |       |       |       |       |
+ - - - + - - - + - - - + - - - + - - - +
|       |       |       |       |       |
|       |       |       |       |       |
|       |       |       |       |       |
+ - - - + - - - + - - - + - - - + - - - +
|       |       |       |       |       |
|       |       |       |       |       |
|       |       |       |       |       |
+ - - - + - - - + - - - + - - - + - - - +
|       |       |       |       |       |
|       |       |       |       |       |
|       |       |       |       |       |
+ - - - + - - - + - - - + - - - + - - - +
|       |       |       |       |       |
|       |       |       |       |       |
|       |       |       |       |       |
+ - - - + - - - + - - - + - - - + - - - +```

**Comment by mdgregor** [comment]
nice work!

**Pull request merged**